packages/twenty-front/src/modules/workflow/workflow-steps/workflow-actions/components/RightDrawerWorkflowSelectActionContent.tsx

```diff
@@ -14,6 +15,14 @@ const StyledActionListContainer = styled.div`
   padding-inline: ${({ theme }) => theme.spacing(2)};
 `;
 
+const StyledSectionTitle = styled.span`
+  color: ${({ theme }) => theme.font.color.tertiary};
+  font-size: ${({ theme }) => theme.font.size.sm};
+  font-weight: ${({ theme }) => theme.font.weight.medium};
+  padding: ${({ theme }) => theme.spacing(2)} ${({ theme }) => theme.spacing(1)}
+    ${({ theme }) => theme.spacing(1)} ${({ theme }) => theme.spacing(1)};
```

Comment:
Maybe we can write it like so:

```suggestion
  padding: ${({ theme }) => `${theme.spacing(2)} ${theme.spacing(1)} ${theme.spacing(1)}} ${theme.spacing(1)}`};
```

I feel it's a _bit_ more readable, but not that much, either 😅

Reply:
I will split the paddings ;)